### ISSUE TITLE
[4.3-dev] trashed tags are not shown in trash

### ISSUE THREAD

**Issue by astridx** [joomla/joomla-cms#39837]
### Steps to reproduce the issue

Create a tag and trash it.

### Expected result

The trashed tag is shown in trash

### Actual result

The trashed tag is not shown in trash. But if you want to create another tag with the same alias, you see the message "Save failed with the following error: A trashed Tag with the same parent tag has the same alias."

### System information (as much as possible)

[systeminfo-2023-02-11T10.56.23+01.00.txt](https://github.com/joomla/joomla-cms/files/10713118/systeminfo-2023-02-11T10.56.23%2B01.00.txt)

**Comment by ReLater** [comment]
I can't confirm using Joomla_4.3.0-beta2-dev (nightly build update package) via "Upload and Update".

**Comment by brianteeman** [comment]
can not confirm
![image](https://user-images.githubusercontent.com/1296369/218257642-960feec3-900b-41b7-a7a9-0404fde02557.png)

**Comment by alikon** [comment]
unable to replicate on a nigthly one

**Comment by ChristineWk** [comment]
can't confirm it also.

Joomla 4.2.8-dev and:

![screen shot 2023-02-11 at 17 38 51](https://issues.joomla.org/uploads/1/94c6c523ece048f48ea22882c59d0072.jpg)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39837">issues.joomla.org/tracker/joomla-cms/39837</a>.</sub>

**Comment by astridx** [comment]
I'm sorry. I can't recreate it now either. I must have done something wrong.

The error occurred during the installation for https://github.com/joomla/joomla-cms/pull/39805, where I changed from branch 4.2-dev to 4.3-dev. 

After changing the branches, I deleted `configuration.php`, ran `npm `and `composer `and reinstalled Joomla. When I then wanted to install the blog sample data, it didn't work and the tags were reported as errors. 

I then used the test sample data for the PR. That worked. 

When I reinstalled everything, the error no longer occurred. Because the data in the PR shows the correct diff to `4.3-dev`, I assume that composer or node did not run correctly this morning and I had missed it.
